scss/responsive.scss

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-width: 1024px) {
```

Comment:
We generally dont use a separated file for responsive content.
You should do this media query inside each responsive scss file